validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -0,0 +1,65 @@
+package hmda.validation
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import hmda.model.fi.SubmissionId
+import hmda.persistence.messages.CommonMessages.{ Command, Event, GetState }
+import hmda.persistence.model.HmdaPersistentActor
+
+object ValidationStats {
+
+  def name = "ValidationStats"
+  case class SubmissionStats(id: SubmissionId, totalLars: Int)
+  case class AddSubmissionStats(stats: SubmissionStats) extends Command
+  case class SubmissionStatsAdded(stats: SubmissionStats) extends Event
+  case class FindTotalLars(institutionId: String, period: String) extends Command
+
+  def props(): Props = Props(new ValidationStats)
+
+  def createValidationStats(system: ActorSystem): ActorRef = {
+    system.actorOf(ValidationStats.props(), "validation-stats")
+  }
+
+  case class ValidationStatsState(stats: Seq[SubmissionStats] = Nil) {
+    def updated(event: Event): ValidationStatsState = event match {
+      case SubmissionStatsAdded(s) =>
+        ValidationStatsState(stats :+ s)
+    }
+  }
+}
+
+class ValidationStats extends HmdaPersistentActor {
+  import ValidationStats._
+
+  override def preStart(): Unit = {
```

Comment:
Do you want to do `super.preStart()` here? (It's something I've seen in other actors)